### PR TITLE
fix: 服务宕机的情况，收不到 response，无法对状态码进行判断，默认业务请求执行失败。

### PR DESCRIPTION
**Pull request by dk-lockdown**
### Issues associated with this PR

服务宕机的情况，收不到 response，无法对状态码进行判断，默认业务请求执行失败。

### CODE REVIEW

**Review by nejisama — DISMISSED** [review]

File: pkg/filter/stream/seata/filter.go [hunk 1]
```diff
@@ -155,6 +155,12 @@ func (f *filter) Append(ctx context.Context, headers api.HeaderMap, buf buffer.I
 						log.Proxy.Errorf(ctx, err.Error())
 					}
 				}
+			} else {
+				// 服务宕机，没有 response，此时 header 为 request header
```
Comment (AlexStocks) [inline pkg/filter/stream/seata/filter.go:159]: translate it into english. thx.

File: pkg/filter/stream/seata/filter.go [hunk 2]
```diff
@@ -172,6 +178,12 @@ func (f *filter) Append(ctx context.Context, headers api.HeaderMap, buf buffer.I
...
```
Comment (AlexStocks) [inline pkg/filter/stream/seata/filter.go:182]: the same as above.

File: pkg/filter/stream/seata/filter.go [hunk 3]
```diff
@@ -156,7 +156,7 @@ func (f *filter) Append(ctx context.Context, headers api.HeaderMap, buf buffer.I
 					}
 				}
 			} else {
-				// 服务宕机，没有 response，此时 header 为 request header
+				// service crash, there is no response，thus, the headers are request headers
```
Comment (AlexStocks) [inline pkg/filter/stream/seata/filter.go:159]: If service crashes, there will be no response. Thus the headers are request headers

**Review by AlexStocks — APPROVED** [review]

**Review by taoyuanyuan — COMMENTED** [review]
lgtm

**Review by taoyuanyuan — DISMISSED** [review]

**Review by AlexStocks — APPROVED** [review]

**Review by AlexStocks — DISMISSED** [review]

**Review by nejisama — APPROVED** [review]

**Review by taoyuanyuan — APPROVED** [review]
lgtm

**Pull request merged**